### DISCUSSION TITLE
slider questions/ oddities

### DISCUSSION THREAD

**Discussion by srolock** [artisan-roaster-scope/artisan#890] [Q&A]
2 questions regarding sliders

1: is it possible to set the initial value after a start for the sliders?
2: Many times (most?) after i press start the gas and fan sliders increment to 100, they dont actually make the output of the action to to 100 but the slider and the number do go to the top and display 100.

Running 2.6.1 on windows 10 pro. It was observed on 2.6.0 as well

Steve

**Comment by poundy** [comment]
Not very knowledgeable on any of this,but thought I'd suggest that you look at the automation options you get from Alarms, particularly for setting a value.

Can I ask what you're controlling, and what your configuration is? That might be relevant for the broad question about your observed (sometimes) slider behaviour.

**Reply by MAKOMO** [reply]
Regarding 1): besides using alarm rules, you can also attach an "Multiple Event" action to the START button which triggers a set of custom buttons (that might be hidden) that do whatever you like.

Regarding 2): by default Artisan does not move any slider on START. Your setup might have some quantifiers defined that move the sliders according to the data Artisan is receiving on the denoted channel. Quantifier actions do not produce any output. You might also have replay-events turned on, which replays events from the loaded background profile after CHARGE. Without knowing your exact setup it remains unclear why things happen the way they do. Sorry.
